Hw5/main.cpp

```diff
@@ -85,27 +82,35 @@ void task4()
 {
 	std::cout << "Enter triangle length: ";
 
-	int length{ 0 };
+	std::size_t length{ 0 };
 	std::cin >> length;
 
-	for (std::size_t i = 0; i < length; ++i)
-		std::cout << std::string(i+1, '*') << '\n';
-
-	std::cout << "\n";
-
-	for (std::size_t i = length; i > 0 ; --i)
-		std::cout << std::string(i , '*') << '\n';
-
-	std::cout << "\n";
-
-	for (std::size_t i = 0; i < length; ++i)
-		std::cout << std::string(i, ' ') + std::string(length, '*') << '\n';
+	//a
+	{
+		for (std::size_t i{ 0 }; i < length; ++i)
+			std::cout << std::string(i + 1, '*') << '\n';
```

Comment:
It's better to use brackets even with one line loop or if

Reply:
I changed it, but brackets with one operation seem excessive and redundant, especially with trivial functions which are not error-prone